static/app/components/events/eventStatisticalDetector/regressionMessage.tsx

```diff
@@ -62,15 +62,26 @@ function EventStatisticalDetectorRegressedPerformanceMessage({
     <DataSection>
       <div style={{display: 'inline'}}>
         {tct(
-          '[detected] Based on the transaction [transactionName], there was a [amount] increase in duration (P95) around [date] at [time]. Overall operation percentage changes indicate what may have changed in the regression.',
+          'Based on the transaction [transactionName], there was a [amount] increase in duration (P95) from [previousDuration] to [regressionDuration] around [date] at [time]. Overall operation percentage changes indicate what may have changed in the regression.',
```

Comment:
Can we also show the absolute delta? So the final message could look like `there was a 123ms (45%) increase in duration`